packages/paste-core/components/button/src/index.tsx

```diff
@@ -29,12 +34,12 @@ const getButtonSize = (variant: ButtonVariants, children: React.ReactNode, size?
   let smartSize: ButtonSizes = 'default';
   if (size != null) {
     smartSize = size;
-  } else if (variant === 'link' || variant === 'destructive_link') {
+  } else if (variant === 'link' || variant === 'destructive_link' || variant === 'reset') {
     smartSize = 'reset';
   } else if (React.Children.count(children) === 1) {
     React.Children.forEach(children, (child) => {
       if (React.isValidElement(child)) {
-        // @ts-ignore
+        // @ts-expect-error we know displayName will exist in React
```

Comment:
Nice use of a newer TS feature! If anyone wants to read more about it, heres some info: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments